### PR TITLE
Add tests for Foreman 2.3 up to 3.2

### LINKED ISSUE #61: Update supported Foreman version
We should test against current Foreman versions and probably also bump the minimum Foreman version (if changes are needed).

### PR DESCRIPTION
**Pull request by laugmanuel**
fixes #61

### CODE REVIEW

**Comment by laugmanuel** [comment]
Integration tests using Capybara are still flaky. I've added a retry mechanism which seems to work around the problems and results in successful tests (sooner or later).

I've opened a new issue to remove the retry and implement a prober fix: https://github.com/dm-drogeriemarkt/foreman_dlm/issues/65

**Comment by laugmanuel** [comment]
Test failures in >= 3.1 are related to https://github.com/dm-drogeriemarkt/foreman_dlm/issues/62 and will be fixed afterwards.

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -0,0 +1,128 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2022-02-23 12:58:32 UTC using RuboCop version 1.10.0.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 1
+# Cop supports --auto-correct.
```
Comment (timogoebel) [inline .rubocop_todo.yml:10]: Should we at least apply the auto correctable fixes here?

Comment (laugmanuel) [reply .rubocop_todo.yml]: I've corrected all safe offenses. I'm not 100% sure if the unsafe ones couldn't cause some problems down the line?

**Review by timogoebel — APPROVED** [review]
LGTM.

**Pull request merged**